### DISCUSSION TITLE
Cannot create property / plugin vite css

### DISCUSSION THREAD

**Discussion by cristianoprogramador** [tailwindlabs/tailwindcss#11545] [Help]
I'm getting this error after i try a new project, i used npm create vite@latest and npm install -D tailwindcss postcss autoprefixer and npx tailwindcss init -p

[plugin:vite:css] [postcss] Cannot create property 'C:\Trabalho\ProjetosPessoais\SoftwareHouse\frontend\tailwind.config.js' on string 'C:/Users/WINDOW~1/AppData/Local/Temp/node-jiti'
C:/Trabalho/ProjetosPessoais/SoftwareHouse/frontend/src/index.css

My files are like this:

`{
  "name": "frontend",
  "private": true,
  "version": "0.0.0",
  "type": "module",
  "scripts": {
    "dev": "vite",
    "build": "tsc && vite build",
    "lint": "eslint src --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
    "preview": "vite preview"
  },
  "dependencies": {
    "react": "^18.2.0",
    "react-dom": "^18.2.0"
  },
  "devDependencies": {
    "@types/react": "^18.0.37",
    "@types/react-dom": "^18.0.11",
    "@typescript-eslint/eslint-plugin": "^5.59.0",
    "@typescript-eslint/parser": "^5.59.0",
    "@vitejs/plugin-react": "^4.0.0",
    "autoprefixer": "^10.4.14",
    "eslint": "^8.38.0",
    "eslint-plugin-react-hooks": "^4.6.0",
    "eslint-plugin-react-refresh": "^0.3.4",
    "postcss": "^8.4.24",
    "tailwindcss": "^3.3.2",
    "typescript": "^5.0.2",
    "vite": "^4.3.9"
  }
}
`

/** @type {import('tailwindcss').Config} */
export default {
  content: [
    "./index.html",
    "./src/**/*.{js,ts,jsx,tsx}",
  ],
  theme: {
    extend: {},
  },
  plugins: [],
}

postcss.config.js:
export default {
  plugins: {
    tailwindcss: {},
    autoprefixer: {},
  },
}

**Comment by mohit2404** [comment]
starting of today I'm also getting same issue didn't found any helpful solution yet.
